roles/django_app_docker/templates/rabbitmq-container.yml.j2

```diff
@@ -0,0 +1,20 @@
+---
+
+container_default_behavior: no_defaults
+name: "{{ django_app_docker_name_prefix }}-rabbitmq"
+image: "{{ django_app_docker_rabbitmq_image }}"
+hostname: "{{ django_app_docker_name_prefix }}-rabbitmq"
+pull: no  # only pull when missing
+restart: "{{ _django_app_docker_env_file.changed }}"
```

Comment:
I think this can be `no` - the RabbitMQ container itself does not depend on the django .env